src/core/internal/hash.d

```diff
@@ -663,7 +663,7 @@ pure nothrow @system @nogc unittest
     version(BigEndian)
     {
         const ubyte[7] a = [99, 4, 3, 2, 1, 5, 88];
-        const uint[2] b = [0x01_02_03_04, 0x05_ff_ff_ff];
+        const uint[2] b = [0x04_03_02_01, 0x05_ff_ff_ff];
```

Comment:
Shouldn't `0x05_ff_ff_ff` be swapped then too?

Reply:
5 bytes are fed into `bytesHash`. `a` is read skipping the first byte, and `b` is read from its start. Both 5-byte sequences should be the same.

The second integer _is_ reversed compared to the little-endian version, which is:
```d
const uint[2] b = [0x04_03_02_01, 0xff_ff_ff_05];
```
The error was that the first integer _shouldn't_ have been reversed, due to a detail of our MurmurHash3 implementation. It reads 4-integer blocks as integers interpreted according to native word order, after which it reads the remaining 0 to 3 bytes in a machine-independent fashion.